app/controllers/v1/tasks_controller.rb

```diff
@@ -54,4 +62,8 @@ def task_params
     params.require(:task).permit(:supplier_id, :framework_id, :status, :due_on,
                                  :period_month, :period_year, :description)
   end
+
+  def replacement_return?
+    params['_jsonapi'].present? && params['_jsonapi']['replacement'].presence.to_s.casecmp('true').zero?
```

Comment:
This was quite hard to understand. Is there a more straightforward way to write this?

Reply:
Which parts were hard to understand?

The existence of the `_jsonapi` param appears to be a quirk of JSONAPI which only became apparent through trial and error, and isn't documented anywhere.

The convoluted way of expressing case-insensitive equality is a rubocop special, I'm afraid. I'd gladly turn that rule off if we can agree that it doesn't really make code better.